### PR TITLE
DTD-2921 Deal with comments from DTD-2735 (Get Pega auth token), toug…

### PR DESCRIPTION
**Pull request by pbyrne84**
…hen up the PEGA update case to expect the right Bearer Token

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ pollingIntervals = 5000
 ```
 
 ### Running the service on a specific port
-```sbt "run 9111"```
+```sbt "run 10003"```
```
Comment (pbyrne84) [inline README.md:20]: Yeah, wrong port. Wild goose chase causing.

File: app/uk/gov/hmrc/debttransformationstub/controllers/AuthController.scala [hunk 2]
```diff
@@ -17,21 +17,68 @@
...
+
+      val maybeClientId = request.body.get("client_id").toList.flatten
+      maybeClientId.headOption match {
+        case Some(actualValue) =>
+          clientIdFileMapping
+            .get(actualValue)
+            .map { fileName =>
+              val file = s"conf/resources/data/auth/$fileName"
+
+              Using(Source.fromFile(environment.getFile(file))) { bufferedSource =>
```
Comment (pbyrne84) [inline app/uk/gov/hmrc/debttransformationstub/controllers/AuthController.scala:50]:
```
Source.fromFile(environment.getFile("conf/resources/data/auth/bearer-token.json")).mkString
```
Doesn't close the resource. Using handles closing etc.

File: conf/resources/data/auth/non-scheduled-bearer-token.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (pbyrne84) [inline conf/resources/data/auth/non-scheduled-bearer-token.json:1]: Created separate files so we can have different auth bearers.

File: README.md (hunk 1 above)
Comment (Aashvin) [reply README.md:20]: Good spot!

**Review by Aashvin — APPROVED** [review]

**Pull request merged**